hermes/crates/cbork/cddl-parser/tests/byte_sequences.rs

```diff
@@ -5,79 +5,36 @@ use cddl_parser::{
     cddl_test::{CDDLTestParser, Parser, Rule},
 };
 
-#[test]
-/// Test if the `HEX_PAIR` rule passes properly.
-fn check_hexpair() {
-    let hex_pairs = vec!["00", "ab", "de", "0f", "f0"];
-
-    let not_hex_pairs = vec!["0", " 0", "0 ", "az", "0p"];
-
-    for hp in hex_pairs {
-        let parse = CDDLTestParser::parse(Rule::HEX_PAIR, hp);
-        assert!(parse.is_ok());
-    }
-
-    for hp in not_hex_pairs {
-        let parse = CDDLTestParser::parse(Rule::HEX_PAIR, hp);
-        assert!(parse.is_err());
-    }
-}
-
-#[test]
-/// Test if the `URL_BASE64` rule passes properly.
-fn check_url_base64() {
```

Comment:
same, we should be able to parse base64 inside the pest parser.  So this test should exist.